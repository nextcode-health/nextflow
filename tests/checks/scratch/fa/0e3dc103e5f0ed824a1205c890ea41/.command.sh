#!/bin/bash -ue
echo Home: $HOME - Input: omega zzz
