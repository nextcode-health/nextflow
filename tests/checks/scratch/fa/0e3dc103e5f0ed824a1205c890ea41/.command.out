Home: /Users/olafurh - Input: omega zzz
